apps/prairielearn/src/sprocs/course_instances_with_staff_access.sql

```diff
@@ -30,7 +30,7 @@ BEGIN
                     ELSE format_date_full_compact(d.end_date, ci.display_timezone)
                 END,
                 'has_course_instance_permission_view',
-                is_administrator OR cip.course_instance_role > 'None'
+                is_administrator OR ia.id IS NOT NULL OR cip.course_instance_role > 'None'
```

Comment:
👀 this (and the other changes in this file) is one of the core updates to auth code; it should be reviewed carefully.